packages/client/src/open-feature.ts

```diff
@@ -87,9 +87,8 @@ export class OpenFeatureAPI extends OpenFeatureCommonAPI {
       this._provider = provider;
       this._providerReady = false;
```

Comment:
Should we also emit an event here?

Reply:
The latest actually specifies that the provider shouldn't be switched until it's initialized, but because of the rapid changes I haven't implemented any actual changes to be in sync with the latest events proposal. I'd rather keep this as a pure refactor and not change any behavior.